src/dtos/measure-dto.ts

```diff
@@ -1,17 +1,22 @@
 import { Measure } from '../entities/dataset/measure';
 
 import { LookupTableDTO } from './lookup-table-dto';
+import { MeasureInfoDTO } from './measure-info-dto';
 
 export class MeasureDTO {
     id: string;
     dataset_id: string;
     fact_table_column: string;
-    join_column: string;
+    join_column: string | null;
```

Comment:
Can this just be `join_column?: string` ?

Reply:
No because in typescript null is different to undefined and join column can be `null`.  This happens after you've done a reset on the measure dimension.